pkg/didcomm/common/service/destination.go

```diff
@@ -19,18 +20,20 @@ type Destination struct {
 	RoutingKeys          []string
 	TransportReturnRoute string
 	MediaTypeProfiles    []string
+	DIDDoc               *did.Doc
 }
 
 const (
-	didCommServiceType = "did-communication"
+	didCommServiceType   = "did-communication"
+	didCommV2ServiceType = "DIDCommMessaging"
 )
 
 // GetDestination constructs a Destination struct based on the given DID and parameters
 // It resolves the DID using the given VDR, and uses CreateDestination under the hood.
-func GetDestination(did string, vdr vdrapi.Registry) (*Destination, error) {
-	docResolution, err := vdr.Resolve(did)
+func GetDestination(didID string, vdr vdrapi.Registry) (*Destination, error) {
```

Comment:
Isn't `didID` var-name redundant ? ie `did` vs `didID`.

Reply:
It looks to me like it was shadowing a package name.

Reply:
that's right